src/shared/components/common/modal/mod-action-form-modal.tsx

```diff
@@ -328,7 +328,7 @@ export default class ModActionFormModal extends Component<
             : "ban_from_community_with_name",
           {
             user: getApubName(this.props.creator),
-            community: getApubName(this.props.community),
+            community: getApubName(this.props.community!),
```

Comment:
Careful with this, it can cause a crash. I'd add a `this.props.community &&` before this block.